src/pages/PlanPage/subRoutes/CancelPlanPage/CancelPlanPage.test.tsx

```diff
@@ -159,6 +160,7 @@ describe('CancelPlanPage', () => {
                 value: planValue,
                 isEnterprisePlan: planValue === Plans.USERS_ENTERPRISEM,
                 isFreePlan: planValue === Plans.USERS_BASIC,
+                isProPlan: planValue === Plans.USERS_PR_INAPPM,
```

Comment:
USERS_PR_INAPPY doesn't matter here? Assuming it doesn't but wanted to ask just in case.

Reply:
nahhh it would tho if we had a test with USERS_PR_INAPPY